packages/shared/src/store/contactActions.ts

```diff
@@ -71,6 +72,145 @@ export async function removeContactSuggestion(contactId: string) {
   }
 }
 
+export async function addContactSuggestions(contactIds: string[]) {
+  // optimistic update
```

Comment:
I would just skip persisting this -- adds complexity + doesn't really net us anything.